web-client/src/types/menu-item.d.ts

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Location from 'react-app-location';
+
+export interface MenuItem {
```

Comment:
Is that right to declare type in here as it's only used by SideNavMenu? Maybe we should export this inside the SideNavMenu.tsx

Reply:
I'm no expert but I think having it declared there makes it easier to find and modify. If we end up using this in another place we can always move it to a more global file

Reply:
You're right. If we need to use these declarations somewhere else then we can consider making them global. It's really better now, thanks!

Done by a126122

Reply:
Just because some this is reused only once doesn’t mean we shouldn’t place it more globally if we expect to want/need it in the future. If things are hidden then they won’t be reused in the future.

Reply:
yup, I agree with this